templates/default/global_conf.erb

```diff
@@ -0,0 +1,31 @@
+# Generated by Chef for <%= node[:fqdn] %>
```

Comment:
Can you remove the 'for <%= node[:fqdn] %>' part here. We've moved away from doing this as a lot of people are using Chef to build AMIs